src/main/java/io/opensaber/providers/SampleViewFunctionProvider.java

```diff
@@ -0,0 +1,31 @@
+package io.opensaber.provider;
```

Comment:
Move this into another package altogether and call it as an example. Say, src/main/java/org/example/provider

Reply:
done